send/ad_group_mu_ucn

```diff
@@ -80,18 +81,20 @@ ldap_unbind($ldap);
 # log results
 ldap_log($service_name, "Added: " . $counter_add . " entries.");
 ldap_log($service_name, "Removed: " . $counter_remove . " entries.");
-ldap_log($service_name, "Updated: " . $counter_update. " entries.");
+ldap_log($service_name, "Updated without errors: " . $counter_update . " entries.");
+ldap_log($service_name, "Updated with errors: " . $counter_updated_with_errors . " entries.");
 ldap_log($service_name, "Failed: " . $counter_fail. " entries.");
 
 # print results for TaskResults in GUI
 print "Added: " . $counter_add . " entries.\n";
 print "Removed: " . $counter_remove . " entries.\n";
-print "Updated: " . $counter_update. " entries.\n";
+print "Updated without errors: " . $counter_update . " entries.\n";
+print "Updated with errors: " . $counter_updated_with_errors . " entries.\n";
 print "Failed: " . $counter_fail. " entries.\n";
 
 $lock->unlock();
 
-if ($counter_fail > 0) { die "Failed to process: " . $counter_fail . " entries.\nSee log at: ~/perun-engine/send/logs/$service_name.log";}
+if ($counter_fail > 0 or $counter_updated_with_errors) { die "Some entries failed to process or some members were not updated.\nSee log at: ~/perun-engine/send/logs/$service_name.log";}
```

Comment:
I would change this to two parts:
```
if ($counter_fail > 0) { die "....";}
if ($counter_updated_with_errors > 0) {die "....";}
```
Message will be more accurate and you will have the info in the printed results.

Reply:
Done